features/javascript-discouraged.yml

```diff
@@ -0,0 +1,43 @@
+name: JavaScript built-ins (legacy)
+description: 
+spec: https://tc39.es/ecma262/multipage/
+group: javascript
+discouraged:
+  according_to:
+    - https://tc39.es/ecma262/#sec-additional-ecmascript-features-for-web-browsers
+    - https://tc39.es/ecma262/#sec-object.prototype-legacy-accessor-methods
+    - https://tc39.es/ecma262/#sec-object.prototype.__proto__
+compat_features:
+  # The following properties are legacy according to:
+  # https://tc39.es/ecma262/#sec-additional-ecmascript-features-for-web-browsers
+  - javascript.builtins.escape
+  - javascript.builtins.unescape
```

Comment:
I think breaking this out into a separate feature (something like "escape() and unescape()") would help us do better discouragement information. We don't have all the data for this yet (it'd be nice to reference and "internal" feature on `javascript` for `encodeURIComponent()` and friends) but I'm thinking ahead here about how a consumer like MDN might be able to use this data for useful developer content (as on [the current MDN page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/escape)).